gengen/util/gengen.go

```diff
@@ -146,8 +146,24 @@ func GenGen(ctx context.Context, cfg *GenesisCfg, cst *hamt.CborIpldStore, bs bl
 		return nil, err
 	}
 
+	emptyMessagesCid, err := cst.Put(ctx, types.MessageCollection{})
```

Comment:
Couldn't this be replaced with types.EmptyMessagesCid?

Reply:
This call is not really about the cid.  We need to put the empty collection to cbor storage so that the node can look it up.  To make sure that we catch issues where gengen and types/messages drift apart in implementation I brought in the explicit check below.

Reply:
👍